### PR TITLE
feat: add drop shadow token

### PR DESCRIPTION
**Pull request by k35o**
## 📝 Description
`dropShadow` could not be granted as Tokens, so it can now be granted.

discussed by https://github.com/chakra-ui/panda/discussions/2677

スクリーンショット 2024-06-19 18 16 07

## ⛳️ Current behavior (updates)
Type error when trying to add `dropShadows` to Tokens

## 🚀 New behavior
- Allow dropShadows to be added to Tokens
- Array and Object are now received and transform into appropriate strings.

## 📝 Additional Information
https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/drop-shadow

### CODE REVIEW

File: packages/token-dictionary/__tests__/transform-drop-shadow.test.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { expect, test } from 'vitest'
+import { TokenDictionary } from '../src/dictionary'
+import { transformDropShadow } from '../src/transform'
+
+test('transform / shadow', () => {
```
Comment (anubra266) [inline packages/token-dictionary/__tests__/transform-drop-shadow.test.ts:5]:
```suggestion
test('transform / dropShadow', () => {
```

**Comment by k35o** [comment]
@anubra266 
fixed test name!

**Comment by anubra266** [comment]
@k35o Thanks for working on this.

To prevent the need for adding new tokens for several properties css has to offer in the future. We've decided to follow the _Closed for modification, open for extension pattern_
https://github.com/chakra-ui/panda/pull/2681

**Comment by k35o** [comment]
@anubra266 
thanks!
That's a good change. I'm looking forward to it.

**Pull request closed without merging**